### DISCUSSION TITLE
Upgrade to io.appium_Java-client 8.3.0

### DISCUSSION THREAD

**Discussion by Londonboy017** [shankybnl/MobileAutomationFramework#33] [General]
Hi everyone! First of all, I want to thank the owner of this GitHub repo for creating and maintaining such a useful resource.

I noticed that the io.appium_Java-client used in this project is 6.1.0 , and I was hoping that someone from the community might be able to help upgrade it to the latest version (8.3.0).

I've attempted to do this myself, but unfortunately, I was unsuccessful. If anyone has experience with upgrading this framework or is interested in taking on this task, it would be greatly appreciated! I believe updating the framework could bring some great improvements and new features to the project.

Thank you in advance for any help you can provide!

**Comment by shankybnl** [comment]
Thank you, @Londonboy017, for the great suggestion. I would love to get thoughts from other folks on the upgrade to start work on it. 

Folks, please share your suggestions.
